helm-charts/yunikorn/templates/ingress.yaml

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{{- if .Values.ingress.enabled -}}
+{{ if .Values.ingress.enabled }}
```

Comment:
Please leave the leading dash to match the same pattern as others (otherwise an extra line is generated). Only the trailing slash should be removed.

Reply:
Others yaml from the helm do not have any dash on first conditional. But indeed removing only the trailing dash is enough to fix ingress deployment and avoid extra line.